presentconjugator.js

```diff
@@ -1,3 +1,4 @@
+
 //Left breakdown function in here so that the conjugate() function is easier to test.
```

Comment:
test code should not be a part of the codebase. If this PR is a WIP please remove it before you send out the final PR